tagcompare/main.py

```diff
@@ -1,10 +1,115 @@
+import argparse
+import sys
+import logging
+
 import capture
 import compare
+import settings
+import logger
+
+
+LOGGER = logger.Logger(name="main", writefile=True).get()
+
+
+def __query_yes_no(question, default="yes"):
```

Comment:
I'd probably look for a module w/ this functionality, though if no high quality ones exist, this looks fine